src/synapse/azext_synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -0,0 +1,212 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import os
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class SynapseScenarioTests(ScenarioTest):
+    location = "northeurope"
+
+    @ResourceGroupPreparer(name_prefix='synapse-cli', random_name_length=16)
+    def test_workspaces(self, resource_group):
+        # create a workspace
+        self._create_workspace()
+
+        # get workspace with workspace name
+        self.cmd('az synapse workspace show --name {workspace} --resource-group {rg}', checks=[])

```

Comment:
check the command result, check other commands as well

Reply:
> check the command result, check other commands as well

added checks